sechub-doc/src/docs/asciidoc/documents/shared/configuration/sechub_config.adoc

```diff
@@ -138,11 +138,16 @@ WARNING: The URI must be whitelisted in your project. Otherwise it will be rejec
 include::sechub_config_example2_webscan_anonyous.json[]
 ----
 <1> Define web scan
-<2> The `URI` to scan. This `URI` must be white listed in `{sechub}` project.
-<3> *Optional*: Define the maximum duration the scan can take. 
+<2> The `URI` to scan. This `URI` must be white listed in `{sechub}` project. Normally without a slash `/` at the end.
+<3> Includes, if you have a special path that is linked nowhere,
```

Comment:
Maybe we should more those also as "Optional" ?. At <5> there is a hint that this entry is optional.
Includes and excludes are AFAIK both optional as well

Reply:
Maybe we should write this like
*Optional*: 
```suggestion
<3> *Optional*: Define includes, if you have a special path that is linked nowhere,
```